config/cops.yml

```diff
@@ -1 +1,24 @@
-Metrics/ClassLength: 400_000
+Metrics/AbcSize:
+  violation_cost: 100_000
```

Comment:
Definitely.

Both are terms and they mean different things. They're related as something like `cost = points / 100_000`.

Clearly, maintaining two values when one can be inferred from the other is kind of dumb, and we're on a pledge to only use "points" from now on.